doc/source/installing.rst

```diff
@@ -110,11 +110,23 @@ Note that all format names are normalized to lower case.
 Building from source
 ++++++++++++++++++++
 
-To build yt from source, you need ``git``, and a C compiler (such as ``gcc``
-or ``clang``).
-
-Then run
-
+There are a couple ways to build yt from source with e.g., ``pip``, all of which
+require a C compiler (such as ``gcc`` or ``clang``).
+
+yt is distributed on PyPI and conda-forge, in the form of pre-built binaries ("wheels").
```

Comment:
Reading that back, it looks like I'm implying conda-forge has wheels, which is incorrect.
```suggestion
yt is primarily distributed on PyPI, in the form of pre-built binaries (wheels).
```